vault/seal/ocikms/ocikms.go

```diff
@@ -225,40 +233,48 @@ func (k *OCIKMSSeal) Encrypt(ctx context.Context, plaintext []byte) (*physical.E
 		return nil, fmt.Errorf("given plaintext for encryption is nil")
```

Comment:
(This is as close as GH's review tool lets me get 😀)

Line 231 can just be: `if len(plaintext) == 0 { `